test/e2e/runs.test.js

```diff
@@ -752,6 +752,7 @@ module.exports = () => {
                     timeO2End: dateValue,
                     timeTrgEnd: dateValue,
                     runQuality: 'test',
+                    lhcPeriod: '123123123',
```

Comment:
Please use the real like example `lhc22_b`

Reply:
Same here